src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,67 @@
+package edu.kis.vh.nursery;
+
+/**
+ * @author 240827
+ *	Default rhymer that works similarly to a stack.
+ */
+public class DefaultCountingOutRhymer {
+
+	public static final int NUMBER = 12;
+	public static final int NEGATIVE = -1;
+	private int[] numbers = new int[NUMBER];
+
+	private int total = NEGATIVE;
+
+	/**
+	 * @return How many numbers have been added to the rhymer so far.
+	 */
+	public int getTotal() {
+		return total;
+	}
+
+	/** 
+	 * Adds number specified in in argument to the rhymer.
+	 * The in argument is not added if the rhymer is full.
+	 * @param in
+	 */
+	public void countIn(int in) {
+		if (!isFull())
+			numbers[++total] = in;
+	}
+
+	/**
+	 * @return boolean value indicating if the rhymer is empty.
+	 */
+	public boolean isEmpty() {
+		return total == NEGATIVE;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.